languages/javalang/builder-renderer/commercetools-java-client/src/main/java/client/NoOpMiddleware.java

```diff
@@ -0,0 +1,19 @@
+package client;
+
+import java.util.concurrent.CompletableFuture;
+
+public class NoOpMiddleware implements Middleware {
```

Comment:
If this is really a singleton, then we should use an `enum` to make it really thread safe.